addons/account/wizard/account_move_send.py

```diff
@@ -365,6 +366,7 @@ def _send_mails(self, moves):
                 subtype_id=subtype.id,
                 partner_ids=partners.ids,
                 attachment_ids=attachment_ids,
+                model_description=model_description
```

Comment:
it's a small detail, but usually we end with a , character (so next time we need to add a line, we only have a one line diff
```suggestion
                model_description=model_description,
```